### PR TITLE
Jshint

### PR DESCRIPTION
**Pull request by travnels**
Added jshint and fixed the issues it identified.

### CODE REVIEW

File: .jshintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (ApfelUser) [inline .jshintignore]: You're missing a white-space here :smile:

Comment (travnels) [inline .jshintignore]: I'm not sure what you mean here.  The node_modules directory is being properly ignored when I run jslint.  Why do I need a white-space?

Comment (ApfelUser) [inline .jshintignore]: Sorry, I meant newline. But that has nothing to do with JSLint, it is so that git can properly display new changes in the future.

**Comment by mashpie** [comment]
thanks an, another issue from my list...

**Pull request merged**